File: forgery/complex-tennis/Randomizer.java
import java.util.Random;

public class Randomizer {
    private Random random;

    public Randomizer( ) {
        this.random = new Random( );
    }

    public boolean LoadedToss( int percentage ){
        double probability  = ( double ) percentage / 100;
        double toss         = this.random.nextDouble( );
        return toss < probability;
    }

    public boolean EqualToss( ){
        return this.random.nextBoolean( );
    }
}

File: forgery/complex-tennis/python/randomizer.py
from random import randint, choice

class Randomizer( object ):
    def __init__( self ):
        super( Randomizer, self ).__init__()

    @staticmethod
    def LoadedToss( probability ):
        toss = randint( 0, 99 )
        return toss < probability

    @staticmethod
    def EqualToss():
        coins = [True, False]
        return choice( coins )
